### DISCUSSION TITLE
Equipo para el Hackaton

### DISCUSSION THREAD

**Discussion by Sebastian-Rob-Cam** [Launch-X-Latam/MisionFrontEnd#582] [Team Posts]
Hola voy un poco tarde pero estoy buscando integrantes para el equipo del hackaton. 😁

**Comment by leofavio** [comment]
Hola. También busco equipo.
lx07546@innovaccion.mx

**Comment by Aa1eexx** [comment]
Igual busco equipo

**Comment by LJavier33** [comment]
Buena tarde, igual un poco tarde en búsqueda de equipo.

**Comment by HoracioTorrez** [comment]
Hola voy atrasado en backend pero este fin me pongo al corriente, si quieren un integrante más me apunto lx15713@innovaccion.mx muchas gracias

**Comment by jlnunezobando** [comment]
Busco equipo lx08651@innovaccion.mx, por favor si me incluyen.

**Comment by Andersonm22** [comment]
aun busca gente para el equipo?, mi correo es lx10407@innovaccion.mx, incluyanme porfa

**Comment by joseandresgavilanes** [comment]
Busco equipo  ***@***.***
